packages/ipa/env/ipa/assets/.gitignore

```diff
@@ -1,2 +0,0 @@
-id_rsa
-id_rsa.pub
```

Comment:
no, there are here for a reason. I am using those keys. Not a good one, I admit. You can add create a issue to fix this another time.

Reply:
created - https://github.com/adaltas/node-nikita/issues/195